src/main/java/loqor/ait/core/lock/LockedDimensionRegistry.java

```diff
@@ -0,0 +1,63 @@
+package loqor.ait.core.lock;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import net.minecraft.item.ItemStack;
+import net.minecraft.item.Items;
+import net.minecraft.world.World;
+import net.minecraft.world.dimension.DimensionTypes;
+
+import loqor.ait.AITMod;
+import loqor.ait.api.TardisEvents;
+import loqor.ait.registry.datapack.SimpleDatapackRegistry;
+
+public class LockedDimensionRegistry extends SimpleDatapackRegistry<LockedDimension> {
```

Comment:
~~Yeah no what's the point of using `Unlockable` without `UnlockableRegistry`~~

UPD: I've read deeper, this is fine